server/src/main/java/org/opensearch/rest/NamedRoute.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.rest;
+
+import org.opensearch.OpenSearchException;
+
+/**
+ * A named Route
+ *
+ * @opensearch.api
```

Comment:
I think this should be:
```suggestion
 * @opensearch.internal
```

Reply:
Changed. Yes this should not directly be used outside of core.